PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeRequest.java

```diff
@@ -52,13 +52,22 @@ public abstract class PayPalNativeRequest implements Parcelable {
     private final ArrayList<PayPalNativeCheckoutLineItem> lineItems;
     private String returnUrl;
     private String userAuthenticationEmail;
+    private final boolean hasUserLocationConsent;
 
     /**
+     * Deprecated. Use {@link PayPalNativeRequest#PayPalNativeRequest(boolean)} instead.
+     *
      * Constructs a request for PayPal Checkout and Vault flows.
      */
+    @Deprecated
     public PayPalNativeRequest() {
+        this(false);
+    }
+
+    public PayPalNativeRequest(boolean hasUserLocationConsent) {
```

Comment:
```suggestion
    PayPalNativeRequest(boolean hasUserLocationConsent) {
```

The deprecated constructor was probably not meant to be public - should a merchant ever construct this themselves?

Reply:
Nope! I agree, we should keep it package private.